docs/installation.rst

```diff
@@ -128,10 +161,11 @@ Alternatively, you can also install a local development copy via `flit`_:
 
 Creating a Conda environment
 ----------------------------
-To create a conda environment including all of `xclim`'s optional and development dependencies, run the following command from within your cloned repo:
+
+To create a conda environment including `xclim`'s dependencies and several optional libraries (notably: `clisops`, `eigen`, `eofs`, and `flox`) and development dependencies, run the following command from within your cloned repo:
 
 .. code-block:: console
 
-    $ conda create -n my_xclim_env python=3.8 --file=environment.yml
+    $ conda env create -n my_xclim_env python=3.8 --file=environment.yml
```

Comment:
```suggestion
    $ conda env create -n my_xclim_env python=3.9 --file=environment.yml
```

Reply:
I was planning on updating this when we decide to move forward on #1268.

Reply:
Ok with me.
My intuition was that as we usually support 3 python versions in parallel, we should suggest the middle one? (Already extensively tested, and not too close to retirement)